charts/prometheus/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: prometheus
 # renovate: github=prometheus/prometheus
 appVersion: v3.4.1
-version: 27.20.1
+version: 27.20.2
```

A new feature is a minor change.

```suggestion
version: 27.21.0
```